Jenkinsfile

```diff
@@ -36,7 +57,7 @@ pipeline {
             }
             steps {
                 sh '''#!/bin/bash -le
-                NAMESPACE=$NAMESPACE BRANCH=$BRANCH make build
+                REGISTRY=$REGISTRY NAMESPACE=$NAMESPACE BRANCH=$BRANCH make build
```

Comment:
This as well why registry is removed

Reply:
I added the registry variable here, I'm guessing its not needed?